packages/voter-dapp/run_tests.sh

```diff
@@ -16,7 +16,6 @@ echo '# 0/16. Prerequisites                                              #'
 echo '#                                                                  #'
 echo -e '####################################################################\n'
 ## Will need to run some JS scripts from packages/core
-cd ../core
```

Comment:
All truffle commands are now run from the voter-dapp directory, so they will use the same artifacts that the voter dapp reads from.